### PR TITLE
VLCMedia: Add missing ParsingOption and copy documentation from libvlc

### PR DESCRIPTION
**Pull request by carolanitz**

### CODE REVIEW

**Review by fkuehne — COMMENTED** [review]
I would rephrase the comment reference to libvlc_dialog_cbs with the VLCKit idioms as libvlc symbols are typically not known to framework users.
Besides that, LGTM!

**Comment by tguillem** [comment]
Fine with me.

**Comment by fkuehne** [comment]
Merged with minor adaptation of the comments as discussed above.
